testsuite/forge/src/backend/local/swarm.rs

```diff
@@ -543,15 +543,14 @@ impl Swarm for LocalSwarm {
     }
 
     fn chain_info(&mut self) -> ChainInfo<'_> {
-        ChainInfo::new(
-            &mut self.root_account,
-            self.validators
-                .values()
-                .map(|v| v.rest_api_endpoint().to_string())
-                .next()
-                .unwrap(),
-            self.chain_id,
-        )
+        let rest_api_url = self
+            .validators()
+            .next()
```

Comment:
Use `.validators()` so that we will be using the endpoint for the first validator deterministically. Otherwise this function will pick one random endpoint which made it hard to test when some nodes are intentionally brought down for testing purpose.